src/models/UserAccess/IUserAccess.ts

```diff
@@ -0,0 +1,23 @@
+export default interface IUserAnnualReportAccess {
+    IsAdmin: boolean;
+    ViewAnnualReportsTable: boolean;
```

Comment:
all boolean variables named wrongly, because they should start with `Is+something`, `Can+do something` and so on

Reply:
Thanks, changed it